README.md

```diff
@@ -420,6 +433,70 @@ Table of Contents
 - [**Forecastie**](https://github.com/martykan/forecastie) <sup>**[[F-Droid](https://f-droid.org/app/cz.martykan.forecastie)]**</sup>
 
 
+------------------------
+
+
+## – Games –
+
+### • Board
+- [**lichess**](https://github.com/lichess-org/lichobile)
+- [**lidraughts**](https://github.com/roepstoep/lidraughts)
+- [**DroidFish**](https://github.com/peterosterlund2/droidfishr) <sup>**[[F-Droid](https://f-droid.org/app/org.petero.droidfish)]**</sup>
+- [**Scid on the go**](https://github.com/gkalab/scidonthego) <sup>**[[F-Droid](https://f-droid.org/app/org.scid.android)]**</sup>
+- [**Open Chaos Chess**](https://github.com/CorruptedArk/open-chaos-chess) <sup>**[[F-Droid](https://f-droid.org/app/dev.corruptedark.openchaoschess)]**</sup>
+- [**Flang**](https://codeberg.org/jannis/FlangAndroid) <sup>**[[F-Droid](https://f-droid.org/app/de.tadris.flang)]**</sup>
+- [**Freebloks**](https://github.com/shlusiak/Freebloks-Android) <sup>**[[F-Droid](https://f-droid.org/app/de.saschahlusiak.freebloks)]**</sup>
+- [**Mill**](https://github.com/calcitem/Sanmill) <sup>**[[F-Droid](https://f-droid.org/app/com.calcitem.sanmill)]**</sup>
+
+
+### • Card
+- [**TriPeaks**](https://github.com/mimoguz/tripeaks-gdx) <sup>**[[F-Droid](https://f-droid.org/app/ogz.tripeaks)]**</sup>
+- [**MTG Familiar**](https://github.com/AEFeinstein/mtg-familiar) <sup>**[[F-Droid](https://f-droid.org/app/com.gelakinetic.mtgfam)]**</sup>
+
+
+### • Educational
+- [**GCompris**](https://invent.kde.org/education/gcompris.git) <sup>**[[F-Droid](https://f-droid.org/app/net.gcompris.full)]**</sup>
+
+
+### • Miscellaneous
+- [**Zoysii**](hhttps://gitlab.com/deepdaikon/Zoysii) <sup>**[[F-Droid](https://f-droid.org/app/xyz.deepdaikon.zoysii)]**</sup>
```

Comment:
There's an extra `h` which breaks the link. I'll fix that up when updating the ToC.